collaboration/wg-installation/README.md

```diff
@@ -0,0 +1,49 @@
+#  Installation Working Group
+
+The purpose of this working group is to:
+* Publicly represent Kyma installation topics.
+* Facilitate collaboration with external contributors.
+* Discuss and work on installation and upgrade improvements/aspects:
+	* Process 
+		* Design - rethink how it should look like
+		* Complexity - straightforward and declarative apporach with limitted manual steps
+		* Time consumption - focus on how long does it take to install/upgrade kyma
```

Comment:
I would expect here also:
* discover and store current baseline for installation time
* monitoring and notifications mechanism about the degradation of installation time